libs/newsletter/driver/src/interfaces/newsletter-service.interface.ts

```diff
@@ -1,10 +1,19 @@
 import { InjectionToken } from '@angular/core';
 import { Observable } from 'rxjs';
 
-import { DaffNewsletterSubmission } from '@daffodil/newsletter';
+import {
+  DaffNewsletterSubmission,
+  DaffNewsletterResponse,
+} from '@daffodil/newsletter';
 
-export const DaffNewsletterDriver = new InjectionToken<DaffNewsletterServiceInterface<DaffNewsletterSubmission, unknown>>('DaffNewsletterDriver');
+/**
+ * The token for the newsletter driver.
+ */
+export const DaffNewsletterDriver: InjectionToken<DaffNewsletterServiceInterface> = new InjectionToken('DaffNewsletterDriver');
 
-export interface DaffNewsletterServiceInterface<T extends DaffNewsletterSubmission, V> {
-  send(email: T): Observable<V>;
+/**
+ * The interface responsible for sending newsletter submissions.
+ */
+export interface DaffNewsletterServiceInterface {
```

Comment:
Can you improve this doc a bit? It would be nice to explain the method, its params, and what other docs tokens would be nice to see for further info. You can look for `@link` or `@see` in the codebase for examples.